app/controllers/api/articles_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Api::ArticlesController < ApplicationController
-  before_action :authenticate_user!, only: [:create]
+  before_action :authenticate_user!, only: %i[create show]
```

Comment:
please chenge this to `[:create, :show]` for readability

Reply:
Fixed!

Reply:
I don't necessarily agree. The `%i[` notation is more modern and more in line with best practices.

Reply:
It is now back to `%i`!